.github/workflows/step_tests.yml

```diff
@@ -39,6 +39,11 @@ jobs:
           pip install -r steps/steps_requirements.txt
           pip install wheel pytest pytest-cov pytest-mock flaky --upgrade
 
+      - name: Install PennyLane-Orquestra
+        run: |
+          python setup.py bdist_wheel
+          pip install dist/PennyLane_Orquestra*.whl
+
```

Comment:
Can use the plugin now here ([see this comment](https://github.com/PennyLaneAI/pennylane-orquestra/pull/6/files#r543933649))